Anytype/Sources/PresentationLayer/Common/SwiftUI/SyncStatus/SyncStatusInfoNetworkIcon.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+import Services
+
+
+struct SyncStatusInfoNetworkIcon: View {
+    @State private var color: Color
+    @ObservedObject var model: SyncStatusInfoViewModel
```

Comment:
Really needed provide parent model?

Reply:
You can use `syncStatusInfo`

Reply:
Updated in this pr
https://github.com/anyproto/anytype-swift/pull/1781